core/src/frame_lifecycle.rs

```diff
@@ -76,6 +76,8 @@ pub enum FramePhase {
 pub fn run_all_phases_avm2(context: &mut UpdateContext<'_, '_>) {
     let stage = context.stage;
 
+    stage.set_invalid(context.gc_context, false);
```

Comment:
I'm not sure we need to do it here, if broadcasting the event already disables it.

Reply:
I put this for situations where the render step is skipped between frames.

Reply:
Hmm, I think that in such a situation we should _not_ disable invalidation. `invalidated` flag should basically mean "we didn't fire the RENDER event yet".
With this line present, there's a risk that someone calls `stage.invalidate()` and the event never actually fires.